src/components/ha-push-notifications-toggle.js

```diff
@@ -2,13 +2,15 @@ import { html } from "@polymer/polymer/lib/utils/html-tag";
 import { PolymerElement } from "@polymer/polymer/polymer-element";
 
 import { getAppKey } from "../data/notify_html5";
+import { isExternal } from "../data/external";
 import { EventsMixin } from "../mixins/events-mixin";
 
 import "./ha-switch";
 
 export const pushSupported =
   "serviceWorker" in navigator &&
   "PushManager" in window &&
+  !isExternal &&
```

Comment:
So this will still just disable the toggle. I think that we should hide the whole row